builtin/builtin.mbti

```diff
@@ -45,7 +45,7 @@ impl Show for ArgsLoc
 
 type Array
 impl Array {
-  append[T](Self[T], Self[T]) -> Unit
+  append[T](Self[T], Self[T], from~ : Int = .., len~ : Int = ..) -> Unit
   binary_search[T : Compare + Eq](Self[T], T) -> Result[Int, Int]
   binary_search_by[T](Self[T], (T) -> Int) -> Result[Int, Int]
   blit_to[A](Self[A], Self[A], len~ : Int, src_offset~ : Int = .., dst_offset~ : Int = ..) -> Unit
```

Comment:
I suggest relax `blit_to` to allow grow automatically

Reply:
@suiyunonghen I have some problem in pushing this branch directly (might be your name is also `main`). Can you clean up your commit message to make it a single commit given the commit is not that large, thanks

Reply:
I deleted the main branch and created a new master as the main branch